openstack/cinder/templates/type-seeds.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Release.IsUpgrade }}
+{{- if .Capabilities.APIVersions.Has "openstack.stable.sap.cc/v1"}}
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: cinder-type-seed
+  labels:
+    component: cinder
+    app: cinder
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    type: seed
+spec:
+  requires:
+  - monsoon3/cinder-seed
+
+  volume_types:
+  - name: vmware
+    is_public: true
+    specs:
+      description: "premium_ssd"
```

Comment:
Reading the code, I don't think this will work:
https://github.com/sapcc/kubernetes-operators/blob/31feb0820940f7b7dad40b7492478307838f5551/openstack-seeder/python/openstack_seeder.py#L2198

https://github.com/sapcc/kubernetes-operators/blob/31feb0820940f7b7dad40b7492478307838f5551/openstack-seeder/pkg/seeder/apis/v1/types.go#L76

It needs to be without `specs`, I guess.

Reply:
I think it should be `extra_specs` and will work then

Reply:
No. `extra_specs` is something else and already used.

Reply:
I see, my bad - description can be used directly without the parent key

Reply:
updated